src/commands/controller.ts

```diff
@@ -0,0 +1,140 @@
+import type yargs from "yargs";
+import type { Arguments, CommandBuilder } from "yargs";
+import { connect, ConnectOptions, ChainName } from "@tableland/sdk";
+import { getWallet } from "../utils";
+
+type Options = {
+  // Local
+  name: string;
+  controller: string;
+
+  // Global
+  privateKey: string;
+  chain: ChainName;
+  alchemy: string | undefined;
+  infura: string | undefined;
+  etherscan: string | undefined;
+};
+
+export const command = "controller <sub>";
+export const desc =
+  "Get, set, and lock the controller contract for a given table";
+
+export const builder: CommandBuilder<Options, Options> = (yargs) =>
+  yargs
+    .command(
+      "get <name>",
```

Comment:
`controller` is a command with three sub-commands, including get, set, and lock.